src/main/scala/pt/tecnico/dsi/openstack/nova/services/Quotas.scala

```diff
@@ -15,28 +15,66 @@ final class Quotas[F[_]: Sync: Client](baseUri: Uri, authToken: Header) extends
 
   private def buildUri(projectId: String, userId: Option[String]): Uri = (uri / projectId).withOptionQueryParam("user_id", userId)
 
+  // In a Rest API if the resource does not exist you return a BadRequest not a NotFound </sarcasm>
+  private def getOption[R: Decoder](uri: Uri): F[Option[R]] = {
```

Comment:
Codacy found an issue: [Do not prefix getters with 'get'](https://app.codacy.com/gh/ist-dsi/scala-nova-client/pullRequest?prid=5937397)